### PR TITLE
OCPBUGS-13346: dont log jwt tokens

### PR DESCRIPTION
**Pull request by ibihim**
# What

- Bump library-go

# Why

- Contains updated audit policies to not log tokens.

### CODE REVIEW

**Comment by stlaz** [comment]
the fixup commit should be squashed

File: go.mod [hunk 1]
```diff
@@ -15,7 +15,7 @@ require (
 	github.com/openshift/api v0.0.0-20230213134911-7ba313770556
 	github.com/openshift/build-machinery-go v0.0.0-20220913142420-e25cf57ea46d
 	github.com/openshift/client-go v0.0.0-20220831193253-4950ae70c8ea
-	github.com/openshift/library-go v0.0.0-20230301092340-c13b89190a26
+	github.com/openshift/library-go v0.0.0-20230707145856-7efc9e03154a
```
Comment (p0lyn0mial) [inline go.mod:18]: ok, this points to the lates commit from the 4.12 branch (https://github.com/openshift/library-go/commits/release-4.12)

**Pull request merged**